spring-pulsar-sample-apps/sample-pulsar-functions/docker-compose.yml

```diff
@@ -0,0 +1,53 @@
+version: '3.5'
+
+services:
+  pulsar:
+    image: kezhenxu94/pulsar:latest
+    command: bin/pulsar standalone
```

Comment:
I'm going to add a comment or a var that can be passed in to use the M1 or regular image. Forgot about that one.

Reply:
The docker file for arm64 has been merged to the Pulsar master. However, the CI/CD pipeline has not yet been upgraded to produce multi-platform docker images. I hope this is done soon, so we can get rid of these workarounds

Reply:
Yep - I saw that. On a similar note I also added support to get rid of the annoying "Mac DSN Netty" warning https://github.com/apache/pulsar/pull/18646.